cortex/templates.py

```
@@ -0,0 +1,588 @@
+                            template = self.load_template(name)
+                            if template:
+                                templates[name] = {
+                                    "name": template.name,
+                                    "description": template.description,
+                                    "version": template.version,
+                                    "author": template.author,
+                                    "type": "user",
+                                    "path": str(template_file)
+                                }
+                        except Exception:
+                            pass
```

Ruff (0.14.5)

`try`-`except`-`pass` detected, consider logging the exception

(S110)